go-sdk/example/handler_test.go

```diff
@@ -11,7 +12,7 @@ import (
 )
 
 func Test_Handler(t *testing.T) {
-	fakeKeptn := sdk.NewFakeKeptn("test-greeting-svc")
+	fakeKeptn := sdk.NewFakeKeptn("test-greeting-svc", &api.APISet{})
```

Comment:
if no api set is required / used, just pass `nil` 
```suggestion
	fakeKeptn := sdk.NewFakeKeptn("test-greeting-svc", nil)
```